docs/execution.md

```diff
@@ -16,7 +16,7 @@ Sending handlers), etc.
 
 The initial `web/index.php` installation queues four middleware decorators:
 
-- _Relay\Middleware\SendingHandler_ to pass along the inbound request without
+- _Relay\Middleware\ResponseSender to pass along the inbound request without
```

Comment:
I see I missed a `_` . Will fix these. Let me know if they are still typos / errors that need to be fixed.